terraform/nowcasting/development/main.tf

```diff
@@ -268,7 +269,11 @@ module "metrics" {
   ecs-task_execution_role_arn = module.ecs.ecs_task_execution_role_arn
   ecs-task_name = "metrics"
   ecs-task_type = "anaylsis"
```

Comment:
Not added in this branch, but typo

Reply:
Fixed with [`fc540f4` (#568)](https://github.com/openclimatefix/ocf-infrastructure/pull/568/commits/fc540f45732bde2171be2fef2d37e1f4875bf6e2)